datafusion/functions-aggregate/src/first_last.rs

```diff
@@ -895,6 +891,31 @@ fn convert_to_sort_cols(
         .collect::<Vec<_>>()
 }
 
+fn replace_order_by_clause(order_by: &mut String) {
```

Comment:
Rather than doing string manipulation here, maybe we could call `create_function_physical_name` to just create the right name in the first place 🤔

Reply:
I see -- `create_function_physical_name` doesn't have sufficient information (`Expr`s etc to do this)

I suppose the alternate is to remember the relevant parts of the expression, but that also seems brittle.

I can't think of anything better at the moment